src/main/scala/natchez/xray/XRayEnvironment.scala

```diff
@@ -0,0 +1,34 @@
+package natchez.xray
```

Comment:
This should ultimately be contributed upstream, but it's here while X-Ray support remains unmerged in natchez.